frontend/src/Charts/AirChart.js

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect, useState } from 'react';
+import {
+  AreaChart,
+  Area,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from 'recharts';
+import axios from 'axios';
+
+const AirChart = ({ dateState, dateFormat }) => {
+  const [linedata, setLineData] = useState([]);
+  const [dataType, setDataType] = useState('DAILY');
+  const [station, setStation] = useState(12);
+  const [isAirHumidity, setAirHumidity] = useState(false);
+  const [isAirPressure, setAirPressure] = useState(false);
+  const [isAirTemperature, setAirTemperature] = useState(true);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const data = {
+          startDate: dateFormat(dateState[0].startDate),
+          endDate: dateFormat(dateState[0].endDate),
+          type: dataType,
+          id: station,
+        };
+        const response = await axios.post('/api/air', data);
+        console.log(response.data);
+        const result = response.data;
+        const mappedResult = result.map((item, index) => {
+          return { ...item, number: index };
+        });
+        setLineData(mappedResult);
+      } catch (err) {
+        console.error('Error during api call:', err);
+      }
+    }
+    fetchData();
+  }, [dataType, station, dateState, dateFormat]);
+
+  return (
+    linedata !== null &&
+    linedata !== undefined && (
+      <>
+        <form>
+          <input
+            type="checkbox"
+            name="AirTemperature"
+            value="airTemperature"
+            onChange={() => setAirTemperature(!isAirTemperature)}
+            checked={isAirTemperature}
+          />
+          <label htmlFor="AirTemperature"> AirTemperature </label>
+          <input
+            type="checkbox"
+            name="AirPressure"
+            value="AirPressure"
+            onChange={() => setAirPressure(!isAirPressure)}
+            checked={isAirPressure}
+          />
+          <label htmlFor="AirPressure"> AirPressure </label>
+          <input
+            type="checkbox"
+            name="AirHumidity"
+            value="AirHumidity"
+            onChange={() => setAirHumidity(!isAirHumidity)}
+            checked={isAirHumidity}
+          />
+          <label htmlFor="AirHumidity"> AirHumidity</label>
+        </form>
+        <div className="container p-3 m-3">
+          <label htmlFor="stationId">Choose a Station:</label>
+          <select
```

Comment:
A selectnek viszont kéne value.
Mint ahogy itt: https://reactjs.org/docs/forms.html#the-select-tag